### PR TITLE
feat:[#332] Alert the user about missing minimum requirements

### LINKED ISSUE #332: Alert the user about missing minimum requirements
Vanilla OS requires at least 4GB of RAM, 2 core CPU. If those requirements are missing, the user should be alerted that there will be issue. A red button should be displayed here to allow the user to proceed.

In the future we could handle this by providing an image with a low footprint and less resources usage (idk how yet).

### PR DESCRIPTION
**Pull request by muhdsalm**
closes #332

### CODE REVIEW

**Comment by Muqtxdir** [comment]
hi @muhdsalm 

thanks for taking initiative on this one, it would be nicer if the host system requirements are shown against the minimum requirements as well
 So that they are aware which component doesn't meet minimum requirements

File: vanilla_installer/gtk/window-cpu.ui [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          <object class="GtkBox">
+            <property name="orientation">vertical</property>
+            <property name="valign">center</property>
+            <property name="spacing">12</property>
+            <child>
+              <object class="GtkLabel" id="title_label">
+                <property name="wrap">True</property>
+                <property name="wrap-mode">word-char</property>
+                <property name="justify">center</property>
+                <property name="label">Not enough CPU cores</property>
```
Comment (MonsterObserver) [inline vanilla_installer/gtk/window-cpu.ui:35]:
```suggestion
                <property name="label">Not Enough CPU Cores</property>
```

File: vanilla_installer/gtk/window-ram.ui [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          <object class="GtkBox">
+            <property name="orientation">vertical</property>
+            <property name="valign">center</property>
+            <property name="spacing">12</property>
+            <child>
+              <object class="GtkLabel" id="title_label">
+                <property name="wrap">True</property>
+                <property name="wrap-mode">word-char</property>
+                <property name="justify">center</property>
+                <property name="label">Not enough RAM</property>
```
Comment (MonsterObserver) [inline vanilla_installer/gtk/window-ram.ui:35]:
```suggestion
                <property name="label">Not Enough RAM</property>
```

**Review by MonsterObserver — CHANGES_REQUESTED** [review]
Titles should use title case, the rest of the UI LGTM

**Review by mirkobrombin — APPROVED** [review]

**Pull request merged**